modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc

```diff
@@ -0,0 +1,552 @@
+
+:description: In this section, we provide guidelines for container orchestration tools suitable for Infinite Scale like {docker-compose-url}[Docker-Compose] and {kubernetes-url}[Kubernetes]. Other container orchestration tools like {docker-swarm-url}[Docker Swarm] can be derived from.
+
+== Introduction
+
+{description} Container orchestration tools are necessary to meet the requirements described in the xref:availability_scaling/availability_scaling.adoc#container[Availability and Scalability] guide starting at the _Container_ section.
+
+The pages
+
+* {swarm-v-kub-1-url}[Docker Swarm vs Kubernetes: how to choose a container orchestration tool] and
+* {swarm-v-kub-2-url}[Kubernetes Vs. Docker Swarm: A Comparison of Containerization Platforms]
+
+can give a brief overview of features and characteristics of both tools.
+
+Note for Kubernetes, ownCloud provides basic {helm-charts-ocis-url}[Helm Charts] that can be used and adjusted.
+
+== Docker Compose
+
+Similar when using _docker run_ and handing over command-line parameters for a single container, you can define a `docker-compose.yml` yaml file which defines all the settings and environment variables for each container in one file. This is the next step when having multi-container environments.
+
+=== Prerequisites
+
+Check if the package `docker-compose` is installed in addition to docker:
+
+[source,bash]
+----
+which docker-compose
+----
+
+If `docker-compose` is installed, you'll be informed. If not, you may get no output at all or a message that it couldn't be found. In that case you need to install `docker-compose` first. On most Linux distributions, you can simply use the package manager to do so. Note that in many cases, this will install a 1.x version (python-based) while a 2.x version (go-based) is preferred. To check which version would be installed with the package manager, simply type:
+
+[source,bash]
+----
+sudo apt-cache policy docker-compose
+----
+
+If the output shows a version like 2.x, you can use the package manager to install docker-compose, like with the following command on Debian-based distributions:
+
+[source,bash]
+----
+sudo apt install docker-compose
+----
+
+If the output shows a version like 1.25.0-1, follow the {docker-compose-install-url}[Install Docker Compose] guide to install a 2.x version.
+
+When done, create a project directory, like `ocis-compose` in your home directory to have a common location for your Infinite Scale compose files.
+
+=== Docker Compose Example
+
+See the {ocis_individual_services-url}[ocis individual services] example which configures services individually using docker-compose, to get a first impression how this can be made. Both, _ocis environment variables_ and _ocis service configuration_ yaml files are used. See the xref:deployment/services/services.adoc[services section] for details of available environment variables and yaml files.
```

Comment:
```suggestion
See the {ocis_individual_services-url}[ocis individual services] example which configures services individually using docker-compose, to get a first impression how this can be done. Both, _ocis environment variables_ and _ocis service configuration_ yaml files are used. See the xref:deployment/services/services.adoc[services section] for details of available environment variables and yaml files.
```